modules/resource-logs-viewing-cli-console.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * logging/resource-logs-viewing.adoc
+
+[id="resource-logs-viewing-cli-console_{context}"]
+= Viewing resource logs
+
+You can view the log for various resources in the {product-title} CLI and web console. Logs read from the tail, or end, of the log.
+
+.Prerequisites
+
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+.Procedure 
+
+To view resource logs:
+
+* Using the web console
+
+. In the {product-title} console, navigate to *Workloads* -> *Pods* or navigate to the pod through the resource you want to investigate. 
++
+[NOTE]
+====
+Some resources, such as builds, do not have pods to query directly. Locate the *Logs* link on the *Details* page.
+====
+
+. Select a project from the drop-down menu.
+
+. Click the pod you want to investigate.
+
+. Click *Logs*. 
+
+* Using the CLI
+
+** View the log for a specific pod:
++
+[source,terminal]
+----
+$ oc logs -f <pod_name> -c <container_name>
+----
++
+Where:
++
+`-f`: Optional: Specifies that the output follows what is being written into the logs. 
++
+`<pod_name>`: Specifies the name of the pod.
++
+`<container_name>`: Optional: Specifies the name of a container. When a pod has more than one container, the container name must be specified.
```

Comment:
I think the formatting for a description list for these variables is a little different based on the docs guidelines:

`----`
code <variable_1>
code <variable_2>
`----`
`+`
where:

<variable_1>:: Specifies the explanation of the first variable.
<variable_2>:: Specifies the explanation of the first variable.

Reply:
Also for present tense: s/the container name must be specified./you must specify the container name./

Reply:
I meant to update the docs guidelines. I believe the way I have it is correct and the guidelines are incorrect in terms of how `asciibinder build` formats it.